configs/setups/awiesm/awiesm-2.1.yaml

```diff
@@ -194,6 +203,20 @@ fesom:
                 make install -j `nproc --all`"
 
         with_wiso: ${general.with_wiso}
+        with_pico: ${general.with_pico}
+        choose_with_pico:
+          True:
+            add_forcing_files:
+              pico_forcing: temp_salt_default
+            add_forcing_sources:
+              # FIXME(PG): Needs to be in a pool somewhere...
+              temp_salt_default: ${general.project_base}/nonstandard_forcings/pico/test.dat
+            add_namelist_changes:
+              namelist.forcing:
```

Comment:
This seems to be a problem; as it is taken out again at the very bottom of the FESOM block. Is there any way around that @mandresm?

Reply:
https://github.com/esm-tools/esm_tools/blob/8436f8784db3c18ca3585b801e3773a66d4e59b6/configs/setups/awiesm/awiesm-2.1.yaml#L315

Reply:
you are right, this needs to be removed. There should be no problem on having the namelist_chnages of the `namelist.forcing` there.